_posts/2020-05-30-solo-socratic-2.md

```diff
@@ -0,0 +1,193 @@
+- [Sort listinvoices and listsendpays by order of creation](https://github.com/ElementsProject/lightning/pull/3752)
+- [Offers preparation](https://github.com/ElementsProject/lightning/pull/3685)
+- [Pay: respect maxfeepercent when choosing a shadow route](https://github.com/ElementsProject/lightning/pull/3693)
+- [Plugin:added invoice creation event](https://github.com/ElementsProject/lightning/pull/3658)
+
+### lnd
+
+- [lnd v0.10.1-beta.rc3](https://github.com/lightningnetwork/lnd/releases/tag/v0.10.1-beta.rc3)
+- [lnd v0.10.0-beta](https://github.com/lightningnetwork/lnd/releases/tag/v0.10.0-beta)
+- [kvdb wrapper for etcd](https://github.com/lightningnetwork/lnd/pull/4015)
+- [add REST endpoints to all RPCs](https://github.com/lightningnetwork/lnd/pull/4251)
+- [walletrpc: add basic coin selection RPCs](https://github.com/lightningnetwork/lnd/pull/4303)
+- [multi: hold keysend payments](https://github.com/lightningnetwork/lnd/pull/4167)
+- [list expired invoices](https://github.com/lightningnetwork/lnd/pull/4168)
+- [dry run migration](https://github.com/lightningnetwork/lnd/pull/2668)
+- [add support for preset close address on inbound channels too](https://github.com/lightningnetwork/lnd/issues/4279)
+- [router+routerrpc: unify sendtoroute and payment responses](https://github.com/lightningnetwork/lnd/pull/4147)
+- [multi: Remove need for DerivePrivKey from watchtower, brontide, netann and sphinx package](https://github.com/lightningnetwork/lnd/pull/4227)
+- [lnrpc: add block height params to GetTransactions and add ListSweeps](https://github.com/lightningnetwork/lnd/pull/4139)
+
+### Wasabi
+
+- [v1.1.11.1 release](https://github.com/zkSNACKs/WalletWasabi/releases/tag/v1.1.11.1)
+- [PayJoin sending support](https://github.com/zkSNACKs/WalletWasabi/pull/3528)
+- [PayJoin through clearnet as fallback](https://github.com/zkSNACKs/WalletWasabi/pull/3637)
+- [Stricten mempool acceptance check to prevent too long mempool chains of CoinJoin](https://github.com/zkSNACKs/WalletWasabi/pull/3670)
+- [Lower maximum CoinJoin coordinator fee now 0.3%](https://github.com/zkSNACKs/WalletWasabi/pull/3502)
+- [HWI upgrade to fix trezor firmware upgrade issue](https://github.com/zkSNACKs/WalletWasabi/pull/3562)
+## rust-lightning
+
+- [C Bindings](https://github.com/rust-bitcoin/rust-lightning/pull/618)
+
+## joinmarket
+
+- [python-bitcointx backend for jmbitcoin](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/536)
+
+## Privacy
+
+- [Design for a CoinSwap Implementation](https://gist.github.com/chris-belcher/9144bd57a91c194e332fb5ca371d0964)
+- [Wasabi Research Club #17: Confidential CoinJoin Construction - Protocol ACL](https://www.youtube.com/watch?v=z3G6CUDPr0c)
+- [Wasabi Research Club #18: Confidential CoinJoin Construction - Protocol KVAC](https://www.youtube.com/watch?v=MvEXPs3KSj8)
+- [WabiSabi CoinJoin protocol cryptographic primitives, currently in peer review](https://github.com/zksnacks/wabisabi/releases)
```

Comment:
i think this is a bit of an overstatement, we're still working towards a 1st draft but we've asked some people for feedback, "peer review" has much stronger connotations about the state of this work